### ISSUE TITLE
[$250] As the approver, I’m receiving the ‘Please approve my…’ email after partially approving a report

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#54667]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.79-5
**Reproducible in staging?:** Needs Reproduction
**Reproducible in production?:** Needs reproduction
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @garrettmknight 
**Slack conversation** (hyperlinked to channel name): [expensify_bugs](https://expensify.slack.com/archives/C049HHMV9SM/p1735576936281949)

## Action Performed:

1. Create a workspace/enable approvals
2. Invite Submitter
3. As submitter, create 2 expenses (they should auto-submit via Instant Submit)
4. As approver, hold one expense and approve the other

## Expected Result:
As the approver, I wouldn’t expect to receive an email about the outstanding expense since it’s on hold
## Actual Result:
As the approver, I received an email asking me to approve the held expense, immediately.
## Workaround:
Unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021877726063406361428</li>
        <li>Upwork Job ID: 1877726063406361428</li>
        <li>Last Price Increase: 2025-01-17</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @jayeshmangwani</details>

**Comment by MelvinBot** [comment]
This has been labelled "Needs Reproduction". Follow the steps here: https://stackoverflowteams.com/c/expensify/questions/16989

**Comment by brunovjk** [comment]
I can reproduce the issue using the [steps described](https://github.com/Expensify/App/issues/54667#issue-2763574363).

<details>
<summary>MacOS: Chrome / Safari</summary>



https://github.com/user-attachments/assets/e93cc5e9-bc82-40f8-90d5-6243bf36abbc

</details>

However, I believe this is not an issue with the frontend.

**Comment by trjExpensify** [comment]
Talking about this [here](https://expensify.slack.com/archives/C049HHMV9SM/p1735576936281949) for the breadcrumbs.

**Comment by garrettmknight** [comment]
This has been reproduced, let's open it up `External` to start.

**Comment by brunovjk** [comment]
> I can reproduce the issue using the [steps described](https://github.com/Expensify/App/issues/54667#issue-2763574363).
> 
> MacOS: Chrome / Safari

I would like to take as C+ if possible. Thank you

**Comment by jayeshmangwani** [comment]
Not overdue, waiting on proposals here

**Comment by jayeshmangwani** [comment]
Same as before https://github.com/Expensify/App/issues/54667#issuecomment-2586855598

cc: @slafortune

**Comment by slafortune** [comment]
Waiting on proposals

**Comment by jayeshmangwani** [comment]
@slafortune I think Issue needs to be fixed on the BE, as we are handling email notifications from there. IMO, we should assign an internal engineer to investigate.

**Comment by trjExpensify** [comment]
@garrettmknight this isn't a bug per se, and isn't it going to be solved by the project you're working on with instant submit changes?

**Comment by garrettmknight** [comment]
Don't think this is actually an instant submit bug, it's more of a hold/approve bug, but I [have a todo to fix it](https://expensify.slack.com/archives/C049HHMV9SM/p1736355537503519?thread_ts=1735576936.281949&cid=C049HHMV9SM) just not super high on my list. We can close and I'll pick up elsewhere.
